Description: Correct two typographical errors.
Forwarded: https://github.com/sdg-mit/gitless/pull/182
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2018-08-25

--- a/gitless/cli/gl_fuse.py
+++ b/gitless/cli/gl_fuse.py
@@ -18,7 +18,7 @@
       'fuse', help=desc, description=(
         desc.capitalize() + '. ' +
         'By default all divergent changes from the given source branch are '
-        'fused. To customize the set of commmits to fuse use the only and '
+        'fused. To customize the set of commits to fuse use the only and '
         'exclude flags'))
   fuse_parser.add_argument(
       'src', nargs='?',
--- a/gitless/core.py
+++ b/gitless/core.py
@@ -1027,7 +1027,7 @@
     """Fuse the given commits onto this branch.
 
     Args:
-      src: the branch (Brach obj) to fuse commits from.
+      src: the branch (Branch obj) to fuse commits from.
       ip: id of the commit to act as the insertion point. The commits to fuse
         are inserted after this commit. ip has to correspond to one of the
         divergent commits from self or the divergent point.
